userspace/falco/app_actions/init_falco_engine.cpp

```diff
@@ -56,35 +56,30 @@ application::run_result application::init_falco_engine()
 
 	configure_output_format();
 
-	// Create "factories" that can create filters/formatters for
-	// syscalls and k8s audit events.
+	// Create "factories" that can create filters/formatters for syscalls
 
 	// libs requires raw pointer, we should modify libs to use reference/shared_ptr
 	std::shared_ptr<gen_event_filter_factory> syscall_filter_factory(new sinsp_filter_factory(m_state->inspector.get()));
-	std::shared_ptr<gen_event_filter_factory> k8s_audit_filter_factory(new json_event_filter_factory());
 
 	// libs requires raw pointer, we should modify libs to use reference/shared_ptr
 	std::shared_ptr<gen_event_formatter_factory> syscall_formatter_factory(new sinsp_evt_formatter_factory(m_state->inspector.get()));
-	std::shared_ptr<gen_event_formatter_factory> k8s_audit_formatter_factory(new json_event_formatter_factory(k8s_audit_filter_factory));
 
 	m_state->syscall_source_idx = m_state->engine->add_source(application::s_syscall_source, syscall_filter_factory, syscall_formatter_factory);
-	m_state->k8s_audit_source_idx = m_state->engine->add_source(application::s_k8s_audit_source, k8s_audit_filter_factory, k8s_audit_formatter_factory);
-
+	
 	if(m_state->config->m_json_output)
 	{
 		syscall_formatter_factory->set_output_format(gen_event_formatter::OF_JSON);
-		k8s_audit_formatter_factory->set_output_format(gen_event_formatter::OF_JSON);
 	}
 
 	for(const auto &src : m_options.disable_sources)
 	{
 		m_state->enabled_sources.erase(src);
 	}
 
-	// XXX/mstemm technically this isn't right, you could disable syscall *and* k8s_audit and configure a plugin.
+	// todo(jasondellaluce,leogr): change this once we attain multiple active source
```

Comment:
This will increase the need to support multiple event sources within one falco process. k8s and syscalls together used to be supported together obviously, and that isn't possible atm.

Reply:
Definitely. I left this comment as a remainder for when we'll work on supporting multiple sources.